vllm/model_executor/model_loader.py

```diff
@@ -83,6 +89,15 @@ def get_model(model_config: ModelConfig, device_config: DeviceConfig,
             initialize_dummy_weights(model)
         else:
             # Load the weights from the cached or downloaded files.
-            model.load_weights(model_config.model, model_config.download_dir,
-                               model_config.load_format, model_config.revision)
+            if model_config.load_format == "tensorizer":
+                # Provide a dynamic load format for `model.load_weights`
+                # to retain tensorizer args from CLI.
+                model_config.load_format = ("tensorizer",
+                                            model_config.tensorizer_args)
+            model.load_weights(
+                model_config.model,
+                model_config.download_dir,
+                model_config.load_format,
+                model_config.revision,
+            )
```

Comment:
This strange workaround was predicated on the fact that `model.load_weights` is defined for every single model class in `vllm/model_executor/models/*.py` 

I sorely wish that their models inherited from some base class because there's seemingly a lot of copy-pasting for their model class methods, but I felt this was the least intrusive way of making this change, rather than adding this option to every single model class and forcing this extra arg to be passed as new models are added, as this is technically a work-around that applies for all model classes extensibly within a few awkward lines of code. 

I very much like your suggestion though so I'll go with that.